include/openssl/comp.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2015-2024 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Rebase issue: unrelated changes. This file was recently moved.

Reply:
I wondered how that got in there... will remove.